Formula/f/fricas.rb

```diff
@@ -0,0 +1,31 @@
+class Fricas < Formula
+  desc "Advanced computer algebra system"
+  homepage "https://fricas.github.io"
+  url "https://github.com/fricas/fricas/archive/refs/tags/1.3.11.tar.gz"
+  sha256 "ce74ad30b2b25433ec0307f48a0cf36e894efdf9c030b7ef7665511f5e6bf7d9"
+  license "BSD-3-Clause"
+  head "https://github.com/fricas/fricas.git", branch: "master"
+
+  depends_on "libice"
+  depends_on "libsm"
+  depends_on "libx11"
+  depends_on "libxau"
+  depends_on "libxdmcp"
+  depends_on "libxpm"
+  depends_on "libxt"
+  depends_on "zstd"
+  depends_on "sbcl"
```

Comment:
```suggestion
  depends_on "sbcl"
  depends_on "zstd"
```

Reply:
Thanks, this is my first attempt, not familiar with the order requirement.  I suspect the linux CI failure is caused by the missing dependency "zstd".  Let's see the CI result after this change.